recipe/yaml-cpp-config.cmake.in.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/yaml-cpp-config.cmake.in b/yaml-cpp-config.cmake.in
+index 7b41e3f..acb82f9 100644
+--- a/yaml-cpp-config.cmake.in
++++ b/yaml-cpp-config.cmake.in
+@@ -5,10 +5,10 @@
+
+ # Compute paths
+ get_filename_component(YAML_CPP_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+-set(YAML_CPP_INCLUDE_DIR "@CONFIG_INCLUDE_DIRS@")
++set(YAML_CPP_INCLUDE_DIR "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@")
+
+ # Our library dependencies (contains definitions for IMPORTED targets)
+ include("${YAML_CPP_CMAKE_DIR}/yaml-cpp-targets.cmake")
+
+ # These are IMPORTED targets created by yaml-cpp-targets.cmake
+-set(YAML_CPP_LIBRARIES "@EXPORT_TARGETS@")
++set(YAML_CPP_LIBRARIES "yaml-cpp")
```

Comment:
Is @EXPORT_TARGETS@ not resolved correctly?

Reply:
`EXPORT_TARGETS` is unpopulated.

The pkg-config file calls it out directly and was the example for my "quick" change: https://github.com/jbeder/yaml-cpp/blob/master/yaml-cpp.pc.in. I imagine the real fix is a mixture of what I did and fixing the export calls in the top level `CMakeLists.txt` file.